_pages/integrations/azure_listeners.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: article
+title: "Event Listeners via Azure SB"
+excerpt-short: "Set up an Event Listener to run an Algorithm with inputs from an Azure Service Bus Queue"
+categories: [integrations]
+tags: [integrations]
+show_related: true
+---
+
+These directions will help you to set up an Algorithmia Event Listener, which will run an algorithm with input payloads provided in an Azure Service Bus Queue. For more info about Azure Service Bus read https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues
+
+This is only available for Enterprise installations of Algorithmia. 
+{: .notice-warning}
+
+## 1. Obtain a template file and account info from Algorithmia
+
+Contact [support@algorithmia.com](mailto:support@algorithmia.com) to obtain the following, which you will need during CloudFormation setup:
+
+* Azure Resource Manager(ARM) template for Service Bus Namespace and Queue
+* Custom Role definition file - QueueReceiver
+* Algorithmia Azure Account
+
+## 2. Invite Algorithmia account as guest account in your account’s Active Directory
+
+Using the Custom Role definition file you have gotten in step 1, create the Custom Role in your account. 
+
+* Authenticate with Azure CLI
+* Run “$ az role definition create --role-definition GuestRole.json”
```

Comment:
Remove the double quotes and replace with backticks. Let's also remove the `$` shell so it's friendly with copy/paste.

```md
Run `az role definition create --role-definition GuestRole.json`
```